### PR TITLE
Add close issue rule

### PR DESCRIPTION
**Pull request by tslogan**
For #5

### CODE REVIEW

**Comment by tslogan** [comment]
Ready for review.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -29,6 +29,9 @@
       of the external developer.
     * Depending on the complexity of the pull request, discretion can be used to
       wait for multiple approvals before merging a pull request.
+* Issue tickets should be closed when a PR has been created to resolve the issue
+  . Typically the issue should be closed when it is completely resolved (PR
+  merged, tested).
```
Comment (amclain) [inline CONTRIBUTING.md]: I think this idea may be able to be expressed in one sentence. How about something like this?

``` text
An issue ticket should only be closed after its corresponding pull request has been merged and tested.
```

(Also, that period is all alone at the beginning of line 33.)

Comment (tslogan) [inline CONTRIBUTING.md]: Updated

**Comment by amclain** [comment]
:+1:

**Comment by tweatherford** [comment]
:+1:

**Pull request merged**